simulator/src/main/java/com/hedera/block/simulator/mode/CombinedModeHandler.java

```diff
@@ -37,28 +37,39 @@ public class CombinedModeHandler implements SimulatorModeHandler {
     private final BlockStreamConfig blockStreamConfig;
 
     /**
-     * Constructs a new {@code CombinedModeHandler} with the specified block stream configuration.
+     * Constructs a new {@code CombinedModeHandler} with the specified configuration.
      *
-     * @param blockStreamConfig the configuration data for managing block streams
+     * @param blockStreamConfig The configuration for block streaming parameters
+     * @throws NullPointerException if blockStreamConfig is null
      */
     public CombinedModeHandler(@NonNull final BlockStreamConfig blockStreamConfig) {
         this.blockStreamConfig = requireNonNull(blockStreamConfig);
```

Comment:
The `blockStreamConfig` field seems to be unused, we could remove?

Reply:
Removed ✅